api/app/responses/api.py

```diff
@@ -52,24 +56,32 @@ def get(self):
         try:
             event = db.session.query(Event).filter(Event.id == args['event_id']).first()
             if not event:
+                LOGGER.warn("Event not found for event_id: {}".format(args['event_id']))
                 return errors.EVENT_NOT_FOUND
 
             form = db.session.query(ApplicationForm).filter(ApplicationForm.event_id == args['event_id']).first()     
             if not form:
+                LOGGER.warn("Form not found for event_id: {}".format(args['event_id']))
                 return errors.FORM_NOT_FOUND
             
             # Get the latest response (note there may be older withdrawn responses)
             response = db.session.query(Response).filter(
                 Response.application_form_id == form.id, Response.user_id == g.current_user['id']
                 ).order_by(Response.started_timestamp.desc()).first()
             if not response:
+                LOGGER.warn("Response not found for event_id: {}".format(args['event_id']))
```

Comment:
Lets make this one an info/debug, because the Application Form tries to get a response every time and we expect many/most of those times to have missing responses, so don't want to generate a bunch of spurious warnings

Reply:
Ah sure thing, didn't realise it would pollute the logs like that.